lib/config.js

```diff
@@ -24,6 +24,9 @@ config['did-io'].methods.v1.cryptoLd = cryptoLd;
 
 const cfg = config['vc-verifier'] = {};
 
+// allow self-signed certificates in dev
+config['https-agent'].rejectUnauthorized = false;
```

Comment:
I don't think this is typically done in default configs in modules because it isn't safe. Can you check to see what the pattern is that we use elsewhere?

Reply:
@dlongley ah, ok. Yeah, I copied this pattern from bedrock-vc-issuer, but I suspect it's wrong there too. Will investigate/fix.

Reply:
Removed from `lib/config.js`